.eleventy.js

```diff
@@ -475,7 +475,7 @@ module.exports = function(eleventyConfig) {
 
         const folderPath = env.page.inputPath
         
-        const widths = [650] // width of blog prose
+        const widths = [450] // width of blog prose
```

Comment:
This should stay at 650, the maximum width an image _can_ display is still 650px wide on non retina (and 1300px on retina).